Realm/RLMResults.mm

```diff
@@ -312,6 +314,109 @@ - (RLMResults *)sortedResultsUsingDescriptors:(NSArray *)properties {
     return [RLMResults resultsWithObjectClassName:self.objectClassName query:move(query) sort:sorter realm:_realm];
 }
 
+namespace {
+    struct RealmCreation {
```

Comment:
I was waiting until https://github.com/realm/realm-cocoa/issues/1584 to properly create a `RealmConfiguration`